index.d.ts

```diff
@@ -0,0 +1,54 @@
+/// <reference types="json-rpc-protocol" />
+/// <reference types="node" />
+
+import { EventEmitter } from "events";
+import { JsonRpcPayload, JsonRpcParamsSchema } from "json-rpc-protocol";
+
+export as namespace JsonRpcPeer;
+
+export * from "json-rpc-protocol";
+
+export class Peer extends EventEmitter implements NodeJS.WritableStream {
```

Comment:
We have no `Peer` exported here.

We only have a default export for `Peer`.

See: https://github.com/JsCommunity/json-rpc-peer/blob/80866319e3ffd13f78994bd40c08ed5fae5decca/src/index.js#L45

I'd like to change the code to the following:

```js
export class Peer ...

export {
  Peer,
}
```

So that we can be able to either `import { Peer } from ...` or `import Peer from ...`

Reply:
Fine by me.